### PR TITLE
Create secrets for Hive cluster pools

### PR DESCRIPTION
**Pull request by hongkailiu**
Required by https://github.com/openshift/release/pull/17364/files

/cc @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 1]
```diff
@@ -2685,3 +2685,52 @@ secret_configs:
...
+        bw_item: registry.connect.redhat.com-pull-secret
+        email_bw_field: email
+        registry_url: registry.connect.redhat.com
+      - auth_bw_attachment: auth
+        bw_item: registry.redhat.io-pull-secret
+        email_bw_field: email
+        registry_url: registry.redhat.io
+  to:
+  - cluster: app.ci
+    name: pull-secret
```
Comment (hongkailiu) [inline core-services/ci-secret-bootstrap/_config.yaml:2715]: The content is exactly as `cluster-secrets-aws` but only the pull secret part.

https://github.com/openshift/release/blob/8554c3bd2bc84b99c4e4fb147448ded4ef4356b2/core-services/ci-secret-bootstrap/_config.yaml#L873-L876

File: core-services/ci-secret-bootstrap/_config.yaml [hunk 2]
```diff
@@ -2685,3 +2685,52 @@ secret_configs:
...
+    namespace: ci-cluster-pool
+    type: kubernetes.io/dockerconfigjson
+- from:
+    aws_access_key_id:
+      bw_item: openshift-ci-pools-hive
```
Comment (hongkailiu) [inline core-services/ci-secret-bootstrap/_config.yaml:2720]: The new account created by DPP.

**Comment by hongkailiu** [comment]
/hold

https://coreos.slack.com/archives/CE3ETN3J8/p1617985337067900

**Comment by hongkailiu** [comment]
/hold cancel

the pull secret in the `install-config-aws-us-east-1.yaml` is removed.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**